spec.md

```diff
@@ -44,9 +44,14 @@ for public re-use.
         │
         └── original          # Intermediate folders are permitted but not necessary
             │                 # and currently contain no extra metadata.
-            └── 0
-                ├── .zarray   # Each label itself is a 5D array matching the highest resolution
-                └── .zattrs   # of the related image and has an extra key, "color", with display information.
+            │
+            └── 0             # Multiscale, labeled image. The name is unimportant but is registered in "labels".
+                ├── .zgroup   # Each labeled image is also a multiscaled image, and therefore a Zarr group.
+                ├── .zattrs   # Metadata of the related image and has an extra key, "color", with display information.
```

Comment:
Does this mean the `"multiscale"` specification is expected here?

Reply:
Yes. This has been updated in the color commits, so we should review whether or not it's clear there.

Reply:
:+1: looking forward to reviewing these